docs/joss_paper/paper.md

```diff
@@ -55,67 +67,67 @@ bibliography: paper.bib
 
-However, popular packages focusing on deep learning methods for medical imaging, such as NiftyNet [@gibson2018niftynet] and MONAI (https://monai.io/), do not yet support image registration, while other open-source projects implementing specific published algorithms, such as the VoxelMorph [@balakrishnan2019voxelmorph], are usually not tested or provided for general research and education purposes.
+However, popular packages focusing on deep learning methods for medical imaging, such as NiftyNet [@gibson2018niftynet] and MONAI (https://monai.io/), do not yet support image registration, while other open-source projects implementing specific published algorithms, such as the VoxelMorph [@balakrishnan2019voxelmorph], are seldom tested or designed for general research and education purposes.
 
-`DeepReg` is a Python package that implements a class of image registration algorithms based on deep neural networks. Both unsupervised algorithms [@de2019deep] and weakly-supervised algorithms [@hu2018label] have been re-implemented. In addition, combining unsupervised losses, such as those derived from image similarity measures, and weak supervision, such as overlap between corresponding anatomical and pathological regions, is at the core of this framework. In this paper, we first briefly describe the major algorithms in `DeepReg`. A summary of the implementation of these algorithms is then provided. The model input and output, with- and without labels, the data sampling methods and the six dataset loaders implementing these are also described. Finally, we demonstrate the versatility of the package by providing a wide range of clinical applications as _DeepReg Demos_ in the `demos` folder, which are also briefly discussed in this paper.
+*DeepReg* is a Python package that implements a class of image registration algorithms based on deep neural networks. Both unsupervised algorithms [@de2019deep] and weakly-supervised algorithms [@hu2018label] have been re-implemented. At the core of this framework is combining unsupervised losses and weak supervision, such as those derived from image similarity measures and overlap between corresponding anatomical and pathological regions of interest (ROIs), respectively. In this paper, we first briefly describe these algorithms available in DeepReg. A summary of the common components implemented for these algorithms is then provided, including the network architectural types, model training and prediction. The six dataset loaders implemented with the supported data sampling approaches are also summarised. Finally, the versatility of the package is demonstrated with a wide range of clinical applications as _DeepReg Demos_ in the "demos" folder, which are also briefly discussed in this paper.
```

Comment:
"At the core of this framework" -> "The core of this framework" ?

Reply:
either reads fine to me.